docs/src/gs/create-workload-cluster.md

```diff
@@ -52,16 +58,20 @@ clusterctl generate cluster <cluster-name>\
 
 ## Create a new workload cluster on bare metal instances using an Ubuntu custom image
 
-Note the addition of `OCI_PV_TRANSIT_ENCRYPTION=false` which is required for most BM shapes.
+Note the addition of the `OCI_CONTROL_PLANE_SHAPE` variables, `OCI_WORKER_SHAPE` variables to change the shape 
+information from the `VM.Standard.E4.Flex` default. You will also need to set`OCI_PV_TRANSIT_ENCRYPTION=false` which is 
```

Comment:
Reword to:

If you are using bare metal shapes, you will also need to set `OCI_PV_TRANSIT_ENCRYPTION=false`